src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+import FooterNav from './FooterNav/FooterNav';
```

Comment:
creating index.js in the component folder should make imports easier. In my opinion it makes a finer structure. In any case we should define a common methodology

Reply:
Can you provide me with an example?

Reply:
We will standardize the component folder at a later date